source/Calamari.GoogleCloudAccounts/GcloudOauthFilesConfiguration.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Calamari.Common.Features.Processes;
+using Calamari.Common.Features.Scripting;
+using Calamari.Common.Plumbing.FileSystem;
+using Calamari.Common.Plumbing.Logging;
+using Calamari.Common.Plumbing.Variables;
+
+namespace Calamari.GoogleCloudAccounts
+{
+    /// <summary>
+    /// Gcloud oauth uses the jwt (as a file on disk) to create another creds config file.
+    /// This file is only used once the first request is made from either the CLI or terraform and is required on disk for the lifetime of the auth session.
+    /// We do not want this file persisted on disk after the session (deployment) is over.
+    /// </summary>
+    public class GcloudOAuthFileConfiguration : IDisposable
```

Comment:
I've added this class to ensure the created files are disposed of, the working directory should always be within the work folder and is automatically disposed of anyway, this is just a fallback.